fixbackend/app.py

```diff
@@ -395,6 +396,17 @@ async def domain_events_swagger_ui_html(req: Request) -> HTMLResponse:
         app.include_router(api_router)
         app.mount("/static", StaticFiles(directory="static"), name="static")
 
+        cookie = cookie_transport(cfg.session_ttl)
+
+        @app.middleware("http")
+        async def refresh_session(request: Request, call_next: Callable[[Request], Awaitable[Response]]) -> Response:
+            response = await call_next(request)
+            if refresh_session_token := request.scope.get("refreshed_session"):
```

Comment:
Can you create a constant for `refreshed_session`? 
So it is easier to see where it is used.